crates/holochain/src/conductor/conductor.rs

```diff
@@ -1531,17 +1531,44 @@ mod app_impls {
                         .flat_map(|c| c.to_be_bytes())
                         .collect();
 
-                    let info = self
-                        .keystore
-                        .lair_client()
-                        .derive_seed(
-                            lair_tag.into(),
-                            None,
-                            dst_tag.into(),
-                            None,
-                            derivation_path.into_boxed_slice(),
-                        )
-                        .await?;
+                    let derive_seed = {
```

Comment:
I've since removed this code, but note that this binds to a closure, so the intent was to call `derive_seed()` to instruct lair to derive the seed, so I think the name makes sense. The new code just duplicates the lair `derive_seed` call instead of creating this closure which turned out not to be succinct, and also confusing.